webapp/src/ts/app.component.ts

```diff
@@ -121,9 +125,12 @@ export class AppComponent implements OnInit {
     private telemetryService:TelemetryService,
     private transitionsService:TransitionsService,
     private ngZone:NgZone,
-    private chtScriptApiService: CHTScriptApiService
+    private chtScriptApiService: CHTScriptApiService,
+    private analyticsModulesService: AnalyticsModulesService,
+    private headerTabsService: HeaderTabsService
```

Comment:
Just a minor note that if you add a trailing comma here, the next person to append more to this list won't need to edit this line as well.

Reply:
Done